### PR TITLE
Added recipe for alembic

### PR DESCRIPTION
**Pull request by pmlandwehr**
Pinging @zzzeek in case he'd like to be added as a maintainer to the `alembic` builder on conda-forge, a library of community-build [`conda`](http://conda.pydata.org) packages. The maintenance burden here is light: all testing and releases are built and deployed automatically with CI. Changes to how the package is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/alembic-feedstock shortly after this PR is finished and merged. If you aren't interested in helping maintain the build, that's entirely fine too.

### CODE REVIEW

File: recipes/alembic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+
```
Comment (jakirkham) [inline recipes/alembic/meta.yaml]: nit: remove extra newline.

File: recipes/alembic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - alembic.script
+    - alembic.testing
+    - alembic.testing.plugin
+    - alembic.util
+
+  commands:
+    - alembic --help
+
+  requires:
+    - nose
```
Comment (jakirkham) [inline recipes/alembic/meta.yaml]: Are we going to run the tests? If not, we probably don't need this.

**Comment by zzzeek** [comment]
I do downstream packaging for Red Hat but that is related to the fact that they pay me a salary :)

File: recipes/alembic/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/alembic/meta.yaml]: done

**Pull request merged**